src/util/gc_byte.rs

```diff
@@ -20,20 +44,32 @@ fn get_gc_byte<VM: VMBinding>(object: ObjectReference) -> &'static AtomicU8 {
     if VM::VMObjectModel::HAS_GC_BYTE {
         unsafe { &*(object.to_address() + VM::VMObjectModel::GC_BYTE_OFFSET).to_ptr::<AtomicU8>() }
     } else {
-        todo!("\"HAS_GC_BYTE == false\" is not supported yet")
+        unreachable!()
     }
 }
 
 /// Atomically reads the current value of an object's GC byte.
 ///
 /// Returns an 8-bit unsigned integer
 pub fn read_gc_byte<VM: VMBinding>(object: ObjectReference) -> u8 {
```

Comment:
Whats the difference between `read_gc_byte` and `get_gc_byte`? Is it possible to remove one of them?

Reply:
I think `get_gc_byte()` is not public. `read/write_gc_byte` uses it.